src/PatternSelect.jsx

```diff
@@ -211,23 +211,31 @@ const PatternSelect = ({letters, onClear, setValue, onSubmit}) => {
 
     const getCoordinates = (index) => {
         const alpha = 2 * Math.PI * index / letters.length 
-        const x =  50 + 39 * Math.cos(alpha)
-        const y = 50 + 39 * Math.sin(alpha)
+        const x =  62.5 + 46 * Math.cos(alpha)
+        const y = 62.5 + 46 * Math.sin(alpha)
+        return {x: x, y: y}
+    }
+
+    const getCircleCoordinates = (index) => {
+        const alpha = 2 * Math.PI * index / letters.length 
+        const x =  62.5 + 63 * Math.cos(alpha)
+        const y = 62.5 + 63 * Math.sin(alpha)
         return {x: x, y: y}
     }
 
     return (
         
     <div style={{display: "flex", justifyContent:"center", padding: 20}}>
-        <svg style={{width: 225, height: 225, background: "var(--c-secondary)", borderRadius: 112.5}} className="patternlock" id="lock" viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Set `borderRadius: 112.5` to `50%` so it remains as circle regardless to change in element `height/width`.